packages/python/examples/flask/requirements.txt

```diff
@@ -0,0 +1,12 @@
+certifi==2022.6.15
+charset-normalizer==2.1.0
+click==8.1.3
+Flask==2.1.2
+idna==3.3
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.1
+../../
```

Comment:
What's this?

Reply:
This is a local install! So installing the readme_metrics package from disk, 2 directories up. I know it looks bad...

Reply:
ah TIL

Reply:
Yeah TIL as well, TIL about Python and the whole pip ecosystem. I'm really glad I use npm day-to-day and not pip.